auction-house/program/Cargo.toml

```diff
@@ -21,24 +21,24 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-solana-program = "1.10"
-anchor-lang = "0.25.0"
-anchor-spl = "0.25.0"
-spl-token = { version = "3.2",  features = ["no-entrypoint"] }
-spl-associated-token-account = {version = "1.0.5", features = ["no-entrypoint"]}
-mpl-token-metadata = { version="1.4", features = [ "no-entrypoint" ] }
+solana-program = "1.14"
+anchor-lang = "0.26.0"
+anchor-spl = "0.26.0"
+spl-token = { version = "3.5",  features = ["no-entrypoint"] }
+spl-associated-token-account = {version = "1.1.1", features = ["no-entrypoint"]}
+mpl-token-metadata = { version="1.7", features = [ "no-entrypoint" ] }
```

Comment:
Should this be token-metadata be 1.8 now?

For that matter, should token-metadata also be updated to use auth rules 1.2.0?  I don't think auth rules changes are significant for this,

But I'm just wondering if you are going through the trouble of updating everything should we put them all at the latest now?

Reply:
It's a moving target tho, if we wait another week or so to merge this PR it can be updated to 1.9 when `Burn` and `Verify` are released. The goal of this PR was to get everything up to Anchor 0.26 , Solana 1.14 and a version of Token Metadata that supports pNFTs. The 1.7 dependency will pull newer minor versions if necessary and any future updates that specifically require it can bump.

Reply:
For example, AH still needs to be updated to support pNFTs so the dependency can be bumped to whatever the current version of Token Metadata is at that point.